browser/src/unocommands.js

```diff
@@ -90,13 +90,12 @@ var unoCommandsArray = {
 	'DefaultNumbering':{global:{context:_('Toggle Ordered List'),menu:_('~Ordered List'),},},
 	'DefineName':{spreadsheet:{menu:_('~Manage...'),},},
 	'DefinePrintArea':{spreadsheet:{menu:_('~Define'),},},
-	'DelBreakMenu':{spreadsheet:{menu:_('Delete Page ~Break'),},},
```

Comment:
I would prefer to not change anything apart of adding new command (it can break existing translations) Could you revert non-related changes?